tests/target/enum.rs

```diff
@@ -27,8 +27,10 @@ enum Bar {
 }
 
 enum LongVariants {
-    First(LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
-          VARIANT),
+    First(
+        LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
+        VARIANT,
+    ),
```

Comment:
Yea, sorry, that's what I meant. So we could format a tuple struct like
```rust
struct Foo(long1,
           long2,
           long3);
```
then?